core/templates/pages/exploration-editor-page/editor-tab/graph-directives/state-graph-visualization.directive.ts

```diff
@@ -177,9 +177,17 @@ angular.module('oppia').directive('stateGraphVisualization', [
                 var dimensions = getElementDimensions();
 
                 // Center the graph at the node representing the current state.
-                origTranslations[0] = (
-                  dimensions.w / 2 - nodeData[$scope.currentStateId()].x0 -
-                  nodeData[$scope.currentStateId()].width / 2);
+                try {
+                  origTranslations[0] = (
+                    dimensions.w / 2 - nodeData[$scope.currentStateId()].x0 -
+                    nodeData[$scope.currentStateId()].width / 2);
+                } catch (error) {
+                  error.message +=
```

Comment:
When writing multiline statements, always wrap inside `(...)` (see [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript).)

Reply:
@kevintab95, would this be fine?